.ci_support/migrations/libprotobuf4233_libgrpc156.yaml

```diff
@@ -1,11 +0,0 @@
-__migrator:
-  build_number: 1
-  kind: version
-  migration_number: 1
-libgrpc:
-- '1.54'
-- '1.56'
-libprotobuf:
-- '3.21'
```

Comment:
Python 3.6 was the last version of Python which was not supporting libprotobuf 4 but was supporting libprotobuf 3. Python 3.6 it is not used anymore on conda-forge.

Hence we can safely drop this version of libprotobuf 3.